tools/release/patch_release.sh

```diff
@@ -0,0 +1,588 @@
+    fi
+
+    if [ "$dry_run" = "false" ]; then
+        echo "Waiting for github actions to propogate..."
+        show_spinner 200
+        # For announce in #help-engineering
+        echo
+        echo "For announcing in #help-engineering"
+        echo "===================================================="
+        echo "Release $target_milestone QA ticket and docker publish"
+        echo "QA ticket for Release $target_milestone " `gh issue list --search "Release QA: $target_milestone in:title" --json url | jq -r .[0].url`
+        echo "Docker Deploy status " `gh run list --workflow goreleaser-snapshot-fleet.yaml --json event,url,headBranch | jq -r "[.[]|select(.headBranch==\"$target_patch_branch\")][0].url"`
+        echo "List of tickets pulled into release https://github.com/fleetdm/fleet/milestone/$target_milestone_number"
+
+        echo 
+    else
+        echo "DRYRUN: Would have printed announce in #help-engineering text w/ qa ticket, deploy to docker link, and milestone issue list link"
+    fi
+
+    next_tag="fleet-v$next_ver"
+    if [ "$dry_run" = "false" ]; then
+        # after passing QA
+        #echo "After QA is passed run these commands"
+        #echo
+        git tag $next_tag
+        git push origin $next_tag
+
+        show_spinner 200
+    else
+        echo "DRYRUN: Would have tagged and pushed $next_tag"
+    fi
+
+    if [ "$dry_run" = "false" ]; then
+        releaser_out=`gh run list --workflow goreleaser-fleet.yaml --json databaseID,event,headBranch,url | jq "[.[]|select(.headBranch==\"$next_tag\")[0]`
+        echo "Releaser running " `echo $releaser_out | jq -r ".url"`
+
+        gh run watch `echo $releaser_out | jq -r ".databaseID"`
+    else
+        echo "DRYRUN: Would found goreleaser action and waited for it to complete"
+    fi
+
+
+    if [ "$dry_run" = "false" ]; then
+        echo "### Changes" > release_notes
+        echo "" >> release_notes
+        cat temp_changelog >> release_notes
+        echo "" >> release_notes
+        echo "### Upgrading" >> release_notes
+        echo "" >> release_notes
+        echo "Please visit our [update guide](https://fleetdm.com/docs/deploying/upgrading-fleet) for upgrade instructions." >> release_notes
+        echo "" >> release_notes
+        echo "### Documentation" >> release_notes
+        echo "" >> release_notes
+        echo "Documentation for Fleet is available at [fleetdm.com/docs](https://fleetdm.com/docs)." >> release_notes
+        echo "" >> release_notes
+        echo "### Binary Checksum" >> release_notes
+        echo "" >> release_notes
+        echo "**SHA256**" >> release_notes
+        echo "" >> release_notes
+        echo '\`\`\`' >> release_notes
+        gh release download $next_tag -p checksums.txt
+        cat checksums.txt >> release_notes
+        echo '\`\`\`' >> release_notes
+
+        echo
+        echo "============== Release Notes ========================"
+        cat release_notes
```

Comment:
next bit of automation will be to have the script just update the release text w/ this file. As of today I don't have a good example to test. Will add in the next iteration that I do.